src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -275,6 +275,34 @@ is the general extensibility point to add additional properties to any activity.
 The `Enrich` option is specific to this instrumentation, and is provided to
 get access to `HttpRequest` and `HttpResponse`.
 
+When overriding the default settings provided by instrumentation or adding
+additional telemetry, it is important to consider the sequence of callbacks.
+Generally, it is recommended to use `EnrichWithHttpResponse` for any activity
+enrichment that does not need access to exceptions, as the instrumentation
+library sets all defaults before this callback. The following is the sequence in
+which these callbacks are executed:
+
+1) Processor `OnStart`
+2) `EnrichWithHttpRequest`
+3) `EnrichWithException`
+4) `EnrichWithHttpResponse`
+5) Processor `OnEnd`
```

Comment:
nit: Would be nice to call out that `SetTag` will override any existing tag of same name, and `SetTag` with null is a way to achieve removing a Tag completely.

Reply:
I added in the example code.